<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="4" passed="2" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-02-14T23:34:59 IST" name="Suite" finished-at="2023-02-14T23:36:03 IST" duration-ms="63556">
    <groups>
    </groups>
    <test started-at="2023-02-14T23:34:59 IST" name="Test" finished-at="2023-02-14T23:36:03 IST" duration-ms="63556">
      <class name="Tests.LoginPageTest">
        <test-method is-config="true" signature="setup()[pri:0, instance:Tests.LoginPageTest@5a5a729f]" started-at="2023-02-14T23:34:59 IST" name="setup" finished-at="2023-02-14T23:35:14 IST" duration-ms="14860" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="verifyInvalidCredentials(java.lang.String,java.lang.String)[pri:0, instance:Tests.LoginPageTest@5a5a729f]" started-at="2023-02-14T23:35:14 IST" name="verifyInvalidCredentials" data-provider="Login Credentials" finished-at="2023-02-14T23:35:16 IST" duration-ms="1956" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[ ]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[admin123]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Error Msg for Blank Username/Password..... expected [Required] but found [Required123]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Error Msg for Blank Username/Password..... expected [Required] but found [Required123]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at Tests.LoginPageTest.verifyInvalidCredentials(LoginPageTest.java:43)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyInvalidCredentials -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:Tests.LoginPageTest@5a5a729f]" started-at="2023-02-14T23:35:16 IST" name="tearDown" finished-at="2023-02-14T23:35:17 IST" duration-ms="453" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setup()[pri:0, instance:Tests.LoginPageTest@5a5a729f]" started-at="2023-02-14T23:35:17 IST" name="setup" finished-at="2023-02-14T23:35:29 IST" duration-ms="12094" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="verifyInvalidCredentials(java.lang.String,java.lang.String)[pri:0, instance:Tests.LoginPageTest@5a5a729f]" started-at="2023-02-14T23:35:29 IST" name="verifyInvalidCredentials" data-provider="Login Credentials" finished-at="2023-02-14T23:35:31 IST" duration-ms="1752" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[Admin]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[ ]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Error Msg for Blank Username/Password..... expected [Required] but found [Required123]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Error Msg for Blank Username/Password..... expected [Required] but found [Required123]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at Tests.LoginPageTest.verifyInvalidCredentials(LoginPageTest.java:43)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyInvalidCredentials -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:Tests.LoginPageTest@5a5a729f]" started-at="2023-02-14T23:35:31 IST" name="tearDown" finished-at="2023-02-14T23:35:31 IST" duration-ms="516" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setup()[pri:0, instance:Tests.LoginPageTest@5a5a729f]" started-at="2023-02-14T23:35:31 IST" name="setup" finished-at="2023-02-14T23:35:42 IST" duration-ms="10969" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="verifyInvalidCredentials(java.lang.String,java.lang.String)[pri:0, instance:Tests.LoginPageTest@5a5a729f]" started-at="2023-02-14T23:35:42 IST" name="verifyInvalidCredentials" data-provider="Login Credentials" finished-at="2023-02-14T23:35:46 IST" duration-ms="3599" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[admin123]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Admin]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyInvalidCredentials -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:Tests.LoginPageTest@5a5a729f]" started-at="2023-02-14T23:35:46 IST" name="tearDown" finished-at="2023-02-14T23:35:46 IST" duration-ms="415" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setup()[pri:0, instance:Tests.LoginPageTest@5a5a729f]" started-at="2023-02-14T23:35:46 IST" name="setup" finished-at="2023-02-14T23:35:59 IST" duration-ms="12547" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="verifyLogin()[pri:0, instance:Tests.LoginPageTest@5a5a729f]" started-at="2023-02-14T23:35:59 IST" name="verifyLogin" finished-at="2023-02-14T23:36:02 IST" duration-ms="3211" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyLogin -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:Tests.LoginPageTest@5a5a729f]" started-at="2023-02-14T23:36:02 IST" name="tearDown" finished-at="2023-02-14T23:36:03 IST" duration-ms="441" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- Tests.LoginPageTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
